packages/venia-concept/src/index.js

```diff
@@ -46,6 +50,25 @@ ReactDOM.render(
     <Adapter apiBase={apiBase} apollo={{ link: apolloLink }} store={store}>
         <AppContextProvider>
             <App />
+            {/* Some extensions require a target */}
+            <Extension targetId="main-ep-before-children">
```

Comment:
What is your opinion about moving this extension creation logic to a file of its own, maybe `SampleExtension.js` and just use `<SampleExtension />` here?

Obviously this is just an example but I am curious to know what you think of it?

My point is, there will be less change in the `venia-concept` related files making things simpler to maintain custom code.